### PR TITLE
Add cookie functionality

### PR DESCRIPTION
**Pull request by PeanutButters93**
Logging in and signing in now stores some user data and the corresponding JWToken in cookies.

To update the code you can use the useCookie hook

To accessing cookies include: use cookie.load e.g.,
cookie.load('userdata')['username']

### CODE REVIEW

File: backend/user_profile_backend/controller/getUser.js [hunk 1]
```diff
@@ -52,7 +52,8 @@ const loginUser = async (request, response) => {
             const user = results.rows[0]
             const isMatch = await bcrypt.compare(password, user.password)  // assuming the password column is named 'password'
             if (isMatch) {
-                const token = jwt.sign({ userId: user.id }, 'yourSecretKey', { expiresIn: '1h' })
+                delete user["password"]
+                const token = jwt.sign({ "userdata": user }, 'yourSecretKey', { expiresIn: '1h' })
```
Comment (zzthian) [inline backend/user_profile_backend/controller/getUser.js]: Can the signing be changed to:
jwt.sign( {"userId": user.id, "isAdmin" : user.is_admin}, ....) 

We only need those two fields

File: backend/user_profile_backend/controller/getUser.js [hunk 2]
```diff
@@ -65,7 +66,8 @@ const loginUser = async (request, response) => {
                     const user = results.rows[0]
                     const isMatch = await bcrypt.compare(password, user.password)
                     if (isMatch) {
-                        const token = jwt.sign({ userId: user.id }, 'yourSecretKey', { expiresIn: '1h' })
+                        delete user["password"]
+                        const token = jwt.sign({ "userdata": user }, 'yourSecretKey', { expiresIn: '1h' })
```
Comment (zzthian) [inline backend/user_profile_backend/controller/getUser.js]: Same here

**Review by zzthian — CHANGES_REQUESTED** [review]
Will need you to demonstrate later during the meeting how things will look in terms of the Application data as seen in Inspect element, and how we can retrieve the token from the cookies as we will need it in all our requests moving forward. But overall looks good. Will approve after changes made and demo seen in meeting

File: backend/user_profile_backend/controller/getUser.js (hunk 1 above)
Comment (PeanutButters93) [reply backend/user_profile_backend/controller/getUser.js]: Changes have been made!

File: backend/user_profile_backend/controller/getUser.js (hunk 2 above)
Comment (PeanutButters93) [reply backend/user_profile_backend/controller/getUser.js]: Changes have been made!

**Review by kohkaixun — APPROVED** [review]
LGTM

**Review by zzthian — APPROVED** [review]
Thanks!

**Pull request merged**